Calibration/TkAlCaRecoProducers/test/DQMRootFileReader.cc

```diff
@@ -83,7 +81,6 @@ void DQMRootFileReader::analyze(const edm::Event &iEvent, const edm::EventSetup
   // NOTE: this is here just because we need it after the beginRun of
   // MEtoEDMCoverter which calls a Reset on all MEs.
   dbe->open(filename, false, "", "", DQMStore::OpenRunDirs::StripRunDirs);
-  dbe->showDirStructure();
```

Comment:
It looks like this module isn't doing much now that the call to `showDirStructure` has been removed.
Is there a suitable replacement?

Reply:
The main point of this module was probably the `dbe->open()`, which loads a file into the `DQMStore`.

However, while the stub is still there, there is currently no code for reading legacy files any more, since there shouldn't be any applications any more:
- A common use in the past were references, but references where removed a while ago.
-  Another potential use might be file format conversions. But converting *from* `TDirectory` ROOT format should never really be required, if anything, we convert *into* that format.

Overall, I don't want to add code for reading this messy format unless there is a really good use case for it.

So, probably this module is just obsolete, but deciding such things is out of scope for this PR (which is giant already).